src/main/java/com/github/ygimenez/method/Pages.java

```diff
@@ -1594,7 +1595,7 @@ public static void addButtons(InteractPage page, Message msg, boolean withSkip,
 
 		List<ActionRow> rows = new ArrayList<>();
 
-		List<Component> row;
+		List<ActionComponent> row;
```

Comment:
Would be better to refrain from using subclass types since the library might support Components other than Button futurely.